src/openrct2/game.h

```diff
@@ -150,6 +150,7 @@ extern bool gInUpdateCode;
 extern bool gInMapInitCode;
 extern sint32 gGameCommandNestLevel;
 extern bool gGameCommandIsNetworked;
+extern char gCurrentLoadedPath[260];
```

Comment:
The only reason why you can not use MAX_PATH is due windows.h include which seems to be easily removed. So maybe this could be merged and afterwards some refactoring to get rid of the include and use the already defined MAX_PATH in platform.h